rest/data/commitq_test.go

```diff
@@ -0,0 +1,37 @@
+package data
+
+import (
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/model/commitq"
+	"github.com/evergreen-ci/evergreen/testutil"
+	"github.com/stretchr/testify/suite"
+)
+
+type CommitQSuite struct {
```

Comment:
I made this a suite because I foresee adding more functionality to the data model (configuring the queue, for instance)